archive/multisource.go

```diff
@@ -162,16 +166,19 @@ func (s *spanSource) ToRequest(req *api.WorkerRequest) error {
 type chunkMultiSource struct {
 	ark      *Archive
 	altPaths []string
+	stats    *ChunkStats
 }
 
+var _ Source = &chunkMultiSource{}
```

Comment:
This is redundant; NewMultiSource wouldn't compile if chunkMultiSource didn't implement Source.